README.md

```diff
@@ -124,13 +124,25 @@ pip install apache-airflow[postgres,google]==1.10.12 \
 
 ## Official source code
 
-Looking at the legal side of the software creation process, Apache Airflow is an Apache Software Foundation (ASF) project. This means that Apache Airflow project releases are source code releases and follow the [ASF Release Policy](http://www.apache.org/legal/release-policy.html). There is only one official release repository for Apache Airflow, like all ASF projects: [The Distribution Site of ASF](https://downloads.apache.org). Official releases are cryptographically signed by the release manager and officially voted on by the PMC members during the [Release Approval Process](http://www.apache.org/legal/release-policy.html#release-approval).
+Apache Airflow is an [Apache Software Foundation](http://www.apache.org) (ASF) project, and our official source code releases:
+
+- Follow the [ASF Release Policy](http://www.apache.org/legal/release-policy.html)
+- Can be downloaded from [the ASF Distribution Directory](https://downloads.apache.org)
+- Are cryptographically signed by the release manager
+- Are officially voted on by the PMC members during the [Release Approval Process](http://www.apache.org/legal/release-policy.html#release-approval)
 
 Following the ASF rules, the source packages released must be sufficient for a user to build and test the release provided they have access to the appropriate platform and tools.
 
-Other ways of retrieving source code are "convenience" methods. For example, tagging in GitHub is a convenient way to mark the git project sources that were used to generate the official source packages. We also have binary "convenience" packages how Airflow is distributed: PyPI releases which are a convenient way of installing Apache Airflow using standard python tools or Docker Images published in the [Apache Airflow DockerHub](https://hub.docker.com/repository/docker/apache/airflow). Those are not official releases but they are built using officially released sources.
+Other ways of retrieving source code are "convenience" methods. For example:
+
+- Tagging in GitHub is a convenient way to mark the git project sources that were used to generate official source packages
+- We also have binary "convenience" packages
+- PyPI releases which are a convenient way of installing Apache Airflow using standard python tools
+- Docker Images published in the [Apache Airflow DockerHub](https://hub.docker.com/repository/docker/apache/airflow).
```

Comment:
```suggestion
- We also have binary "convenience" packages
    - PyPI releases which are a convenient way of installing Apache Airflow using standard python tools
    - Docker Images published in the [Apache Airflow DockerHub](https://hub.docker.com/repository/docker/apache/airflow).
```
How about nesting in this way?

Reply:
good point - just pushed a commit that i think improves